Makefile.one

```diff
@@ -45,6 +47,21 @@ checklib_%:
 LAST_GIT_TAG ?= $(shell $(CMD_GIT) describe --tags --match 'v*' 2>/dev/null)
 VERSION ?= $(if $(RELEASE_TAG),$(RELEASE_TAG),$(LAST_GIT_TAG))
 
+# os version
+
+OS_RELEASE_FILE ?= /etc/os-release
+
+OS_ID_LIKE := $(shell $(CMD_GREP) -m1 'ID_LIKE=' $(OS_RELEASE_FILE) | $(CMD_CUT) -d= -f2 | $(CMD_SED) 's/\"//g')
+OS_VERSION_ID := $(shell $(CMD_GREP) -m1 'VERSION_ID=' $(OS_RELEASE_FILE) | $(CMD_CUT) -d= -f2 | $(CMD_SED) 's/\"//g')
+
+ifneq (,$(findstring rhel,$(OS_ID_LIKE)))
+        RHEL_MAJOR := $(shell echo $(OS_VERSION_ID) | cut -d. -f1)
+        RHEL_MINOR := $(shell echo $(OS_VERSION_ID) | cut -d. -f2)
+        RHEL_RELEASE_CODE := $(shell echo $$(($(RHEL_MAJOR) << 8 | $(RHEL_MINOR))))
+else
+        RHEL_RELEASE_CODE := 0
+endif
+
```

Comment:
Why do we need this?
CO-RE doesn't require this field (it is now guarded by #ifndef CORE in bpf code), and if the bpf object is compiled with RHEL headers, it already exists.
Not only this, I think we can totally revert #896 (and modifications made later related to RHEL and CORE).

Reply:
I haven't realized we had it protected by the ifdef CORE changes, indeed. Dropping this commit...